docs/docs/layouts/inline.md

```diff
@@ -85,4 +89,50 @@ Define the featured image you would like to use for the module.
 
 ![Image Inline Module](../examples/img/layouts/inline/image.png)
 
-<pre data-src="../../examples/src/layouts/inline/image.js"></pre>
\ No newline at end of file
+<pre data-src="../../examples/src/layouts/inline/image.js"></pre>
+
+## background image
```

Comment:
This should be `backgroundImage` the titles are formatted as they appear in the config.